wsdlgo/encoder.go

```diff
@@ -1369,24 +1369,26 @@ func (ge *goEncoder) genGoOpStruct(w io.Writer, d *wsdl.Definitions, bo *wsdl.Bi
 	name := goSymbol(bo.Name)
 	function := ge.funcs[name]
```

Comment:
`function := ge.funcs[name]` should be `function := ge.funcs[bo.Name]` and the whole issue is fixed.
No other changes are required.

Reply:
No, this fix doesn't seem to work with other wsdls, unfortunately :(